### PR TITLE
Feature/update api

### PR DESCRIPTION
**Pull request by JavaJens**
This PR updates the API, and adds AVG Cadence/HR values.
Furthermore, an issue of missing distance information is fixed.

@liskin This is WIP, as I haven't tested this on the bike, just indoors.

### CODE REVIEW

File: src/main/scala/cz/nomi/locusRflktAddon/display/Pages.scala [hunk 1]
```diff
@@ -284,6 +286,8 @@ object Const {
     val elevationDownhillWorkout = "ELEVATION_WORKOUT_DOWNHILL"
     val cadenceCurrent = "BIKE_CAD_CURRENT"
     val heartRateCurrent = "HR_CURRENT"
+    val averageCadenceWorkout = "CADENCE_WORKOUT_AVG"
+    val averageHrmWorkout = "HRM_WORKOUT_AVG_MOVE"
```
Comment (liskin) [inline src/main/scala/cz/nomi/locusRflktAddon/display/Pages.scala:290]: Why does hrm have _MOVE but cadence not?
(I'd expect that this depends on whether "record only when moving" is checked in Locus. Is that not so?)

Comment (JavaJens) [inline src/main/scala/cz/nomi/locusRflktAddon/display/Pages.scala:290]: I guess I just copied it badly...
I'll check the API

**Comment by liskin** [comment]
I cherry-picked the two commits except the distance one (that was an error on Locus side which is fixed now). I renamed a few things slightly. Thanks a lot!

**Comment by JavaJens** [comment]
I think we should look into getting the distance via the TrackStats anyway, as the TrackRecordContainer is deprecated.

File: src/main/scala/cz/nomi/locusRflktAddon/display/Pages.scala (hunk 1 above)
Comment (JavaJens) [inline src/main/scala/cz/nomi/locusRflktAddon/display/Pages.scala:290]: I don't know how they're recorded. I'd suggest removing the _MOVE suffix...

**Comment by liskin** [comment]
I committed a change that avoids TrackRecordContainer entirely and gets distance via TrackStats.getTrackLength. Not sure it works, though, I'll test it outside later today.

**Comment by JavaJens** [comment]
Oh, didn't see this. Nevermind then :)

**Pull request closed without merging**